packages/webapp/src/utils/helpers.js

```diff
@@ -263,7 +263,7 @@ const getDeveloperEligibilityResults = session => {
 
 const formatAppRef = appRef => `${appRef.substr(0, 3)}-${appRef.substr(3, 3)} ${appRef.substr(6, 3)} ${appRef.substr(9, appRef.length)}`
 
-const formatSortCode = sortCode => `${sortCode.substr(0, 2)} ${sortCode.substr(2, 2)} ${sortCode.substr(4, 2)}`
+const formatSortCode = sortCode => `${sortCode.substring(0, 2)} ${sortCode.substring(2, 4)} ${sortCode.substring(4, 7)}`
```

Comment:
Last bit should be `${sortCode.substring(4, 6)` not 7 (although this technically works as it's just going to the end of the string with 7)

Reply:
Fixed